package.json

```diff
@@ -109,6 +114,9 @@
     "vite": "^6.1.0",
     "vitest": "3.0.7"
   },
+  "peerDependencies": {
+    "@deck.gl/carto": "~9.1.0"
+  },
```

Comment:
I don't think we need this as a peer dependency – or at least not for this PR. And adding it will bring in dependencies on much of deck.gl. Could we wait to add this until we're sure we need it, so we can still publish standalone releases in the meantime?

Reply:
Yep, you're right - I was using Javi's test app as a starting point and copied this over by accident